.github/scripts/tests/kind-integration.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
Before it was easy to distinguish the different steps because they would appear in different sections in the GH Action. Now they are all clumped together, making it difficult to tell apart. 

Can you add log messages with user-friendly spacing to separate the different stages out in the logs. This will help with troubleshooting/debugging in the future.

Reply:
+1, brief comments for every section would be nice to have.

Reply:
True. 

Do you mean like logging the comment with some `-----`?
Example:

**Current code**
```
# Build image
( cd $GIT_WORKSPACE && make podman-build -e IMG="${DSPO_IMAGE}" )
```

**Proposed change**
```
echo "---------------------------------"
echo "Build image"
( cd $GIT_WORKSPACE && make podman-build -e IMG="${DSPO_IMAGE}" )
```

Reply:
```
echo "---------------------------------"
echo "Build image"
echo "---------------------------------"
( cd $GIT_WORKSPACE && make podman-build -e IMG="${DSPO_IMAGE}" )
```
how abou this?

Reply:
Done